src/transformState.js

```diff
@@ -5,7 +5,42 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type } = action;
+
+    switch (type) {
+      case 'addProperties':
```

Comment:
Would be great to create variables for string values to avoid errors due to typos.